TelCo.ColorCoder/ColorFromPairNumber.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace TelCo.ColorCoder
+{
+    /// <summary>
+    /// This class provides the mapping of color to pair number.
+    /// </summary>
+    class ColorFromPairNumber

```

Comment:
Class can be reorganized, this has a single static method without any class members. Do you need this class?

Reply:
I wanted to keep both ColorFromPairNumber and PairNumberFromColor in same class as ColorMapOperations but the restriction on line count of 50 made me to split it